make transfer less of a requirement (#864)
ropensci/dev_guide: softwarereview_intro.Rmd

- Your package README will feature a peer-review badge linking to the software review thread.
- Your package will get a [`docs.ropensci.org` docs website](#rodocsci) that you can link from DESCRIPTION.

> - If desired, your package repo may be transferred to the rOpenSci organization.

Author: For PT translation https://github.com/ropensci/dev_guide/pull/852/files#r2018747592

Reviewer: Hi @maelle ! This is updated here: https://github.com/ropensci/dev_guide/pull/822/files > softwarereview_intro.pt.Rmd > line 47.

Author: Thank you!!

> - Your package repo will be transferred to the rOpenSci organization if you agree.

Reviewer 2: ```suggestion
- If desired, your package repo may be transferred to the rOpenSci organization.
```
